packages/components/package.json

```diff
@@ -1,14 +1,45 @@
 {
-  "name": "@puik/components",
+  "name": "@prestashopcorp/puik-components",
```

Comment:
proposal for a new naming : "@prestashopcorp/puik-vue-components" ?

Reply:
According to another comment before : @aAmorim27 doesn't agree with this. Maybe you could discuss about that together.